README.md

```diff
@@ -1,4 +1,4 @@
-# Weave - the Docker network
+# Weave - weaving applications into containers
```

Comment:
Yes, reading it now. I'll delete this branch and redo it. Sorry.

On Fri, Jun 5, 2015 at 1:44 PM, Michael Bridgen <notifications@github.com>
wrote:

> In README.md
> <https://github.com/weaveworks/weave/pull/851#discussion_r31808880>:
>
> > @@ -1,4 +1,4 @@
> > -# Weave - the Docker network
> > +# Weave - weaving applications into containers
>
> Isn't it supposed to be "weaving containers into applications" (i.e., a
> transposition)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/weave/pull/851/files#r31808880>.
>



-- 
fintan@weave.works